### PR TITLE
feat: add option to enable/disable open port to public checkbox

### PR DESCRIPTION
**Pull request by adrysn**
This PR resolves https://github.com/lablup/backend.ai-console/issues/752.

Internal ticket: OP#838.

### CODE REVIEW

File: src/components/backend-ai-login.ts [hunk 1]
```diff
@@ -316,6 +317,11 @@ export default class BackendAILogin extends BackendAIPage {
     } else {
       this.allow_project_resource_monitor = true;
     }
+    if (typeof config.general === "undefined" || typeof config.general.openPortToPublic === "undefined" || config.general.allowProjectResourceMonitor === '' || config.general.openPortToPublic == false) {
```
Comment (inureyes) [inline src/components/backend-ai-login.ts]: config.general.allowProjectResourceMonitor === '' should be  config.general.openPortToPublic === ''

**Review by inureyes — CHANGES_REQUESTED** [review]
config.general.allowProjectResourceMonitor === '' need to be modified. (maybe typo)

File: src/components/backend-ai-login.ts (hunk 1 above)
Comment (adrysn) [reply src/components/backend-ai-login.ts]: Oops... Thanks. I pushed with typo fixed.

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**